teamwork/apps/profiles/views/EditProfileView.py

```diff
@@ -157,19 +158,18 @@ def edit_profile_helper(request, username):
         Helper function that saves profile information from the ProfileForm
     """
 
-    if request.user.profile.isGT:
```

Comment:
We might want to revert this back, and instead pull `tempProfile`, also rename to something along the lines of `User`